### PR TITLE
Remove the implementation of showing the old QR code

### PR DESCRIPTION
**Pull request by Sachin-Mamoru**
### Purpose

**Previous UI**

image

**New UI**

image

With the new suggested change, we’ll be using refresh icon to generate the new qr code and won’t be showing the old qr code now. This behaviour is similar to backup code generation as well.

### Checklist

- [ ] e2e cypress tests locally verified. (for internal contributers)
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Relevant backend changes deployed and verified
- [ ] [Unit tests](/docs/testing/UNIT_TESTING.md) provided. (Add links if there are any)
- [ ] [Integration tests](https://github.com/wso2/product-is/tree/master/modules/integration) provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines
- [ ] Ran FindSecurityBugs plugin and verified report
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets

### CODE REVIEW

**Comment by Achintha444** [comment]
### Design Review

- Do we need to have the yellow box in the modal ? IMO it is not required, the modal is enough.
- Did we get the content in the modal reviewed by the content team ?

File: apps/myaccount/src/components/multi-factor-authentication/authenticators/totp-authenticator.tsx [hunk 1]
```diff
@@ -713,6 +680,49 @@ export const TOTPAuthenticator: React.FunctionComponent<TOTPProps> = (
             });
     };
 
+    /**
+     * Render the totp QR code regenerate confirmation modal
+     */
+    const renderConfirmRegenerateModal = (): ReactElement => {
+        return (
```
Comment (Achintha444) [inline apps/myaccount/src/components/multi-factor-authentication/authenticators/totp-authenticator.tsx:687]:
```suggestion
    const renderConfirmRegenerateModal = (): ReactElement => {
```

File: apps/myaccount/src/components/multi-factor-authentication/authenticators/totp-authenticator.tsx [hunk 2]
```diff
@@ -713,6 +680,49 @@ export const TOTPAuthenticator: React.FunctionComponent<TOTPProps> = (
...
+            <Modal
+                data-testid={ `${testId}-regenerate-confirm-modal` }
+                size="small"
+                open={ isConfirmRegenerationModalOpen }
+                closeOnDimmerClick={ false }
+                dimmer="blurring"
+            >
+                <Modal.Content data-testid={ `${testId}-regenerate-confirm-modal-content` }>
```
Comment (Achintha444) [inline apps/myaccount/src/components/multi-factor-authentication/authenticators/totp-authenticator.tsx:695]: refrain from using `data-testid` use `data-componentid` instead.

Comment (Achintha444) [reply apps/myaccount/src/components/multi-factor-authentication/authenticators/totp-authenticator.tsx:695]: check other places

Comment (Sachin-Mamoru) [reply apps/myaccount/src/components/multi-factor-authentication/authenticators/totp-authenticator.tsx:695]: This is a component we had earlier, not a new one. So it has already used testids and I think if we change them, a lot of e2e might break.

File: apps/myaccount/src/components/multi-factor-authentication/authenticators/totp-authenticator.tsx [hunk 3]
```diff
@@ -724,59 +733,15 @@ export const TOTPAuthenticator: React.FunctionComponent<TOTPProps> = (
...
-                        </Segment>
                         <div className="ml-3 mr-3">
-                            <Message className="display-flex" size="small" warning>
-                                <Icon name="warning sign" color="orange" corner />
-                                <Message.Content className="tiny">
-                                    { t(commonConfig.accountSecurityPage.mfa.totp.regenerateWarning) }
-                                </Message.Content>
-                            </Message>
-                            {
-                                commonConfig.accountSecurityPage.mfa.totp.showRegenerateConfirmation
```
Comment (JayaShakthi97) [inline apps/myaccount/src/components/multi-factor-authentication/authenticators/totp-authenticator.tsx:741]: Let's remove the `showRegenerateConfirmation` from config files as well. With the improved UI this config won't be used.
https://github.com/wso2/identity-apps/blob/f5b6df60ae32bd48e851d7173124793f780979e7/apps/myaccount/src/extensions/configs/common.ts#L39
After the unification the config is always set to `false`. Therefore no point of keeping it at all.

File: apps/myaccount/src/components/multi-factor-authentication/authenticators/totp-authenticator.tsx [hunk 4]
```diff
@@ -724,59 +733,15 @@ export const TOTPAuthenticator: React.FunctionComponent<TOTPProps> = (
...
-                            { qrCode
-                                ? <QRCode value={ qrCode } data-testid={ `${ testId }-view-modals-scan-qrcode` }/>
-                                : null
-                            }
-                        </Segment>
                         <div className="ml-3 mr-3">
-                            <Message className="display-flex" size="small" warning>
-                                <Icon name="warning sign" color="orange" corner />
-                                <Message.Content className="tiny">
-                                    { t(commonConfig.accountSecurityPage.mfa.totp.regenerateWarning) }
```
Comment (JayaShakthi97) [inline apps/myaccount/src/components/multi-factor-authentication/authenticators/totp-authenticator.tsx:737]: Check the possibility of removing `regenerateWarning` config as well. Seems the usage has been removed this PR.

File: apps/myaccount/src/components/multi-factor-authentication/authenticators/totp-authenticator.tsx (hunk 2 above)
Comment (Achintha444) [reply apps/myaccount/src/components/multi-factor-authentication/authenticators/totp-authenticator.tsx:694]: better to add a `data-componentid` param then, keeping the `data-testid`, and create an issue to check the E2E on this

**Comment by Sachin-Mamoru** [comment]
This will be addressed via a different approach, hence closing the PR.

**Pull request closed without merging**